packages/web-app-files/src/components/SideBar/Shares/Links/DetailsAndEdit.vue

```diff
@@ -226,15 +230,15 @@ export default defineComponent({
     }
   },
   computed: {
+    currentLinkRole() {
+      return LinkShareRoles.getByBitmask(parseInt(this.link.permissions), this.isFolderShare)
+    },
```

Comment:
iThe parseInt here looks a bit misplaced. Again it would be cool to have a type/interface for the link - then parsing would probably happen in a more appropriate place? 

I see that you are just moving code but now might be a good time for a little cleanup while you touch the code anyway